installation/install.sh

```
@@ -2574,6 +2583,188 @@ install_morph_server(){
   systemctl is-enabled morph &>/dev/null && systemctl restart morph || true
 }
 
+license_server_installer_bin() {
+    local license_server_dir="$ZSTACK_INSTALL_ROOT/$CATALINA_ZSTACK_CLASSES/ansible/license-server"
+    local installer_bin=''
+
+    [ ! -d "$license_server_dir" ] && return 0
+    installer_bin=`find "$license_server_dir" -maxdepth 1 -type f -name "zstack-license-server.${BASEARCH}.bin" | head -n 1`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)